data_model/src/lib.rs

```diff
@@ -286,7 +286,18 @@ pub type ValueBox = Box<Value>;
 
 /// Sized container for all possible values.
 #[derive(
-    Debug, Clone, PartialEq, Eq, Decode, Encode, Deserialize, Serialize, FromVariant, IntoSchema,
+    Debug,
+    Clone,
+    PartialEq,
+    Eq,
+    Decode,
+    Encode,
+    Deserialize,
+    Serialize,
+    FromVariant,
+    IntoSchema,
+    PartialOrd,
```

Comment:
I'm not sure if total ordering of the catch-all Value type makes sense, but I'm guessing that we need this for another upstream type?